artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/proton/ProtonServerSenderContext.java

```diff
@@ -754,23 +759,11 @@ private int writeHeaderAndAnnotations(LargeBodyReader context,
    }
 
    private void finishLargeMessage() {
-      lmUsageDown();
```

Comment:
why did you remove this.. this was the whole point of the fix I recently added.

Reply:
Sorry I have removed by accident while reading of main, will fix this tomorrow

Reply:
fixed!